pkg/controller/provider/controller.go

```diff
@@ -157,6 +160,28 @@ func (r Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (r
 					_ = r.DB().Close(true)
 				}
 			}
+
+			// Wait for the PV to release
+			time.Sleep(5 * time.Second)
+
+			// List all PVs with a specific prefix and label
+			labelSelector := labels.SelectorFromSet(labels.Set{"subapp": "ova-server"})
+			pvList := &v1.PersistentVolumeList{}
+			err = r.Client.List(context.TODO(), pvList, &client.ListOptions{
+				LabelSelector: labelSelector,
+			})
+			if err != nil {
+				r.Log.Error(err, "Failed to list PVs")
+			} else {
+				for _, pv := range pvList.Items {
+					if strings.HasPrefix(pv.Name, "ova-server-pv-") && pv.Status.Phase == v1.VolumeReleased {
```

Comment:
the fact that the PV is released means that its reclaim policy is 'retained'
it should be 'delete' and then, since it's not correlated with a storage class, the PV would remain in failed state

Reply:
posted #685 for this

Reply:
ack, before the deletion I didn't want to leave failed undressed PV in the system so I switch it to release, but now it will be deleted  a few second so we can change the policy.